cli/admin/src/test/java/io/pravega/cli/admin/readerGroup/ParseReaderGroupStreamCommandTest.java

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.cli.admin.readerGroup;
+
+import io.pravega.cli.admin.AdminCommandState;
+import io.pravega.cli.admin.utils.TestUtils;
+import io.pravega.client.ClientConfig;
+import io.pravega.shared.security.auth.DefaultCredentials;
+import io.pravega.test.common.SecurityConfigDefaults;
+import io.pravega.test.integration.utils.SetupUtils;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.Before;
+import org.junit.After;
+import org.junit.rules.Timeout;
+
+import java.io.File;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
+
+import static io.pravega.test.integration.utils.TestUtils.pathToConfig;
+
+
+/**
+ *  This test is for testing the segment store cli commands.
+ */
+public abstract class ParseReaderGroupStreamCommandTest {
+    // Setup utility.
+    protected static final SetupUtils SETUP_UTILS = new SetupUtils();
+    protected static final AtomicReference<AdminCommandState> STATE = new AtomicReference<>();
+    protected static final int CONTAINER_COUNT = 1;
+
+    private static final String ENDPOINT = "localhost";
+    private static final String FILENAME = "output/file";
+
+    @Rule
+    public final Timeout globalTimeout = new Timeout(60, TimeUnit.SECONDS);
+
+    private ClientConfig clientConfig;
+
+    public void setup(boolean enableAuth, boolean enableTls) throws Exception {
+        ClientConfig.ClientConfigBuilder clientConfigBuilder = ClientConfig.builder().controllerURI(SETUP_UTILS.getControllerUri());
```

Comment:
SETUP_UTIL.getControllerUri() returns a stale URI if invoked before `SETUP_UTILS.startAllServices`. ref: https://github.com/pravega/pravega/blob/6a3e934f65e152060d51775a4f732ae835088707/test/integration/src/main/java/io/pravega/test/integration/utils/SetupUtils.java#L159

Please ensure all the properties are set up correctly before running the test. The AdminCommandState is used to generate the controller URI inside the ParseReaderGroupStreamCommand. We should ensure it points to the correct port and not the default port.